kernels/rv64gv/3/bli_sgemm_rv64gv_asm_16x4.S

```diff
@@ -0,0 +1,419 @@
+#define REALNAME bli_sgemm_rv64gv_ker_16x4
```

Comment:
`m4` macros are more powerful than C preprocessor macros.

Just a style thing at this point.